### PR TITLE
Pilot 6155: add logic to display `TRASHED` item in `file list` command

### PR DESCRIPTION
**Pull request by colorzzr**
## Summary

in order to delete items in trashbin, a new logic will need to be add into `file list` command for listing items in trash bin. The example will be:

```
$ poetry run python app/pilotcli.py file list tp01/trash
test20240911(greenroom)  test20240607(greenroom)  chrome_100_percent.pak(core)
```

## JIRA Issues

Pilot 6155

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Refactor or reformatting

## Testing

Are there any new or updated tests to validate the changes?

- [x] Yes
- [ ] No

## Test Directions

add new test for listing items in trashbin

### CODE REVIEW

**Review by andricDu — APPROVED** [review]

File: app/services/file_manager/file_list.py [hunk 1]
```diff
@@ -23,35 +25,48 @@ def __init__(self):
         super().__init__(AppConfig.Connections.url_bff)
 
         self.endpoint = AppConfig.Connections.url_bff + '/v1'
+        self.zone_map = {
+            0: AppConfig.Env.green_zone,
+            1: AppConfig.Env.core_zone,
+        }
```
Comment (afmam) [inline app/services/file_manager/file_list.py]: I think we should place this logic in app config to make it easier to maintain. We did this in bff-web here: https://github.com/PilotDataPlatform/bff-web/blob/develop/app/config.py#L147-L150

**Review by afmam — CHANGES_REQUESTED** [review]
Left one comment

File: app/services/file_manager/file_list.py (hunk 1 above)
Comment (colorzzr) [reply app/services/file_manager/file_list.py]: thanks will update it!

Comment (afmam) [reply app/services/file_manager/file_list.py]: Thank you!

**Review by afmam — APPROVED** [review]
LGTM

**Pull request merged**